part-2-server/server.js

```diff
@@ -0,0 +1,70 @@
+var http = require('http')
+var fs = require('fs')
+
+var server = http.createServer(function(request, response) {
+        var url = request.url
```

Comment:
Nice! This will work. It is a little inefficient because it will need to do a for loop for every request...instead, if the url starts with 'poem' you could have split the url into two substrings, one for 'poem-' and the other for the number, which you could then to look up the correct file. But the fact that it works is the most important thing!